lib/benchmarks/README.md

```diff
@@ -0,0 +1,10 @@
+# Devicelab Tests
+
+This directory, `lib/benchmarks/`, is used for performance testing for the Gallery on Web.
```

Comment:
I think the benchmarks directory would be better placed at the top level, keeping lib for the app source.

Reply:
I think I can only put it inside `lib`.

The benchmark is not run from inside this folder, but from `macrobenchmarks` inside `flutter/flutter`; please see https://github.com/flutter/flutter/pull/57576

`macrobenchmarks` needs to import the gallery and use the `GalleryAutomator` class, so this file needs to stay inside `lib`.

Reply:
I see, the gallery is imported as a package, so everything needs to be in lib. Sounds good